octomap/octomap-config.cmake.in

```diff
@@ -12,21 +12,32 @@
 #
 #
 #    This file will define the following variables:
+#      - OCTOMAP_FOUND          : TRUE
 #      - OCTOMAP_LIBRARIES      : The list of libraries to links against.
 #      - OCTOMAP_LIBRARY_DIRS   : The directory where lib files are. Calling
 #                                 LINK_DIRECTORIES with this path is NOT needed.
 #      - OCTOMAP_INCLUDE_DIRS   : The OctoMap include directories.
+#      - OCTOMAP_MAJOR_VERSION  : Major version.
+#      - OCTOMAP_MINOR_VERSION  : Minor version.
+#      - OCTOMAP_PATCH_VERSION  : Patch version.
+#      - OCTOMAP_VERSION        : Major.Minor.Patch version.
 #
 # ===================================================================================
 
 @PACKAGE_INIT@
 
+set(OCTOMAP_FOUND TRUE)
```

Comment:
I'd rather have this separately tracked from this PR in #138 (same for the other ones)